sdk-workflows/src/main/java/io/dapr/workflows/WorkflowContext.java

```diff
@@ -153,24 +152,24 @@ default <V> Task<V> waitForExternalEvent(String name, Class<V> dataType) {
    * @param <V>        the expected type of the activity output
    * @return a new {@link Task} that completes when the activity completes or fails
    */
-  <V> Task<V> callActivity(String name, Object input, TaskOptions options, Class<V> returnType);
+  <V> Task<V> callActivity(String name, Object input, WorkflowExecutionOptions options, Class<V> returnType);
```

Comment:
Was there a reason you preferred `Execution` over `Task`? Even if its a durable task phrase, it signifies a `basic unit of work` and `are the tasks that get orchestrated in the business process` according to our [dapr docs here](https://docs.dapr.io/developing-applications/building-blocks/workflow/workflow-features-concepts/#workflow-activities).  I would prefer to keep it as `TaskOptions` or `WorkflowTaskOptions` for [parity reasons with dotnet](https://github.com/dapr/dotnet-sdk/blob/release-1.15-rc01/src/Dapr.Workflow/WorkflowContext.cs#L105)

```suggestion
  <V> Task<V> callActivity(String name, Object input, WorkflowTaskOptions options, Class<V> returnType);
```